sources/packages/web/src/services/http/dto/EducationProgramOffering.dto.ts

```diff
@@ -79,6 +79,10 @@ export class EducationProgramOfferingAPIInDTO {
   @Expose()
   offeringType: OfferingTypes;
   @Expose()
+  isAviationOffering: AviationYesNoOptions;
+  @Expose()
+  aviationCredentialType?: AviationCredentialTypeOptions;
```

Comment:
`offeringWILComponentType` is defined as `string`, any particular reason to define a type for the `AviationCredentialTypeOptions`?